README.md

```diff
@@ -33,15 +34,25 @@ and
 *or if you're using Gradle:*
 
 ```bash
-implementation 'com.rudderstack.sdk.java:rudderanalytics-client:1.0.0'
+implementation 'com.rudderstack.sdk.java:rudderanalytics-client:1.0.1'
 ```
 
 ## Initialize ```RudderClient```
 ```java
+FlushBlock flushBlock = FlushBlock.create(); // Initialization in order to block further method invocation until the flush completes. 
```

Comment:
Please update the snippets according to our implementations

Reply:
It is updated accordingly